resources/views/docs/_structures/build.md

```diff
@@ -33,6 +33,7 @@ Field             | Type                                | Notes
 ------------------|-------------------------------------|------
 changelog_entries | [ChangelogEntry](#changelogentry)[] | If the build has no changelog entries, a placeholder is generated.
 versions          | [Versions](#build-versions)         | |
+youtube_id        | string                              | |
```

Comment:
The field isn't optional but the value may be null

Reply:
good catch, thanks. I have it marked null locally as well.

Reply:
should still be moved to the main table instead of optional